misc/scripts/install-local.sh

```diff
@@ -1095,9 +1094,19 @@ function fail_out_functions() {
 
 function run_function() {
     local func="$1"
+
+    echo "#! /bin/bash" | sudo tee "$func.tmp" > /dev/null
```

Comment:
We should also randomize the file name, for better security. Maybe with [mktemp](https://linux.die.net/man/1/mktemp)